src/plugins/mask.ts

```diff
@@ -0,0 +1,4 @@
+import Vue from 'vue';
+import VueTheMask from 'vue-the-mask';
```

Comment:
Could we use `Maska ` plugin from wallet?
https://github.com/beholdr/maska

Reply:
totally agree